src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -23,6 +23,11 @@ public interface IFunctionDescriptor
         string DisplayName { get; }
 
         /// <summary>
+        /// Name to create node
+        /// </summary>
```

Comment:
we need to put this property into the interface to be able to use it in `CreationName` property:
https://github.com/DynamoDS/Dynamo/pull/3855/files#diff-34cffe0b0daeae46fcac66d874d7fdbbR31